pep-0688.rst

```diff
@@ -293,10 +293,41 @@ should deprecate and eventually remove it.
 Backwards Compatibility
 =======================
 
-As the runtime changes in this PEP only add new functionality, there are
-no backwards compatibility concerns.
+``__buffer__`` and ``__release_buffer__`` attributes
+----------------------------------------------------
 
-However, the recommendation to remove the special behavior for
+As the runtime changes in this PEP only add new functionality, there are
+few backwards compatibility concerns.
+
+However, code that uses a ``__buffer__`` or ``__release_buffer__`` attribute for
+other purposes may be affected. While all dunders are technically reserved for the
+language, it is still good practice to ensure that a new dunder does not
+interfere with too much existing code, especially widely used packages. A survey
+of publicly accessible code found:
+
+- PyPy `supports <https://doc.pypy.org/en/latest/__pypy__-module.html#generally-available-functionality>`__
+  a ``__buffer__`` method with compatible semantics to those proposed in this
+  PEP. A PyPy core developer `expressed his support <https://discuss.python.org/t/pep-688-making-the-buffer-protocol-accessible-in-python/15265/34>`__
+  for this PEP.
+- pyzmq `implements <https://github.com/zeromq/pyzmq/blob/fe18dc55516ef50d168fc02f8550a67ff5b5633d/zmq/backend/cffi/message.py#L190>`__
+  a PyPy-compatible ``__buffer__`` method.
+- mpi4py `defines <https://github.com/mpi4py/mpi4py/blob/453b87d0da37c5914b91afb511b188556dff2a9c/src/mpi4py/typing.py#L66>`__
+  a ``SupportsBuffer`` protocol that would be equivalent to this PEP's ``collections.abc.Buffer``.
+- numpy used to have an undocumented behavior where it would access a ``__buffer__`` attribute
+  (not method) to get an object's buffer. This was `removed <https://github.com/numpy/numpy/pull/13049>`__
+  in 2019 for numpy 1.17. The behavior would have last worked in numpy 1.16, which only supported
```

Comment:
```suggestion
- NumPy used to have an undocumented behavior where it would access a ``__buffer__`` attribute
  (not method) to get an object's buffer. This was `removed <https://github.com/numpy/numpy/pull/13049>`__
  in 2019 for NumPy 1.17. The behavior would have last worked in NumPy 1.16, which only supported
```

---

Another point in favour: the NEP 29 recommendation by the Scientific Python community is for NumPy 1.21+. They recommended dropping 1.16 in Jan 2021:

https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table